plugins/headlamp-plugin/plugin-management/multi-plugin-management.js

```diff
@@ -116,6 +116,7 @@ class MultiPluginManager {
       this.pluginsDir === '' ? undefined : this.pluginsDir, 
       this.headlampVersion === '' ? undefined : this.headlampVersion, 
       this.progressCallback === null ? undefined : this.progressCallback,
+      null, // signal
```

Comment:
This commit "fix param bug" should be merged in with the commit that added this code.

To make the commit logs cleaner, as it's not necessary to have a separate commit for this. Since it hasn't been merged into the main branch yet.